modules/sandboxed-containers-scheduling-workloads.adoc

```diff
@@ -3,5 +3,60 @@
 // * sandboxed_containers/deploying_sandboxed_containers.adoc
 
 [id="sandboxed-containers-scheduling-workloads_{context}"]
+= Scheduling {sandboxed-containers-first} workloads
 
-= Scheduling sandboxed containers workloads
+You can schedule your workloads to run on {sandboxed-containers-first}.
+
+.Prerequisites
+
+* You have {product-title} {product-version} installed on your cluster.
+* You have installed the OpenShift CLI (`oc`).
+* You have access to the cluster as a user with the `cluster-admin` role.
```

Comment:
Shouldn't the `admin` role (as opposed to `cluster-admin`) be sufficient here? @zanetworker @jensfr what do you think?

Reply:
@c3d - OCP documentation uses cluster-admin as the role. However, if that's the not the _actual_ role then we need to update to whatever the actual name of the role is.

Reply:
@c3d @sagidlow  
I'm not even sure we need to mention the exact required role. This is about adding the runtimeClassName to your pod-templated resources. If you're a developer you have access to your projects and the correct role to modify your deployment configs etc. And if you're cluster-admin you can do everything anyway. 

I think for someone to be able to modify or create a Pod/Deployment even the 'edit' role would be good enough. But again, I think be should be less specific and write something like: "You have access to the project as a user with a role that can create and modify pod-templated resources"